src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx

```diff
@@ -7,21 +7,29 @@ import { Paragraph } from "../../../typography/Paragraph"
 const ContentPageHeader = ({
   title,
   summary,
+  lastUpdated,
   breadcrumb,
   buttonLabel,
   buttonUrl,
   LinkComponent,
 }: ContentPageHeaderProps) => {
   return (
-    <div className="bg-site-primary-100 px-4 py-8 lg:px-5 lg:py-16">
+    <div className="bg-site-primary-100 px-4 py-8 md:py-10 lg:px-5 lg:py-16">
```

Comment:
issue: I think it should be `px` that should be changed?

Reply:
Confirmed with Se Hyun - have fixed it in https://github.com/isomerpages/isomer-components/pull/88/commits/c4a7878aec9f2a41c0a3522627dbd824d4d446b8!